### ISSUE TITLE
simple.erl example does not work

### ISSUE THREAD

**Issue by tisba** [HdrHistogram/hdr_histogram_erl#22]
I just noticed that the example is broken on master. I'm on OS X 10.10.4, using Erlang/OTP 17 [erts-6.4].

With the current master (https://github.com/HdrHistogram/hdr_histogram_erl/commit/5570a5a345454b4aab0c8c7b161b2f194dc27cad) I get the following error when running `examples/simple.erl`:

```
Runtime: 4.250542secs 2352641.14553rps
escript: exception error: bad argument
  in function  hdr_histogram:print_csv/1
     called as hdr_histogram:print_csv(<<>>)
  in call from simple_erl__escript__1437__226631__273844:main/1 (./examples/simple.erl, line 26)
  in call from escript:run/2 (escript.erl, line 752)
  in call from escript:start/1 (escript.erl, line 276)
  in call from init:start_it/1
  in call from init:start_em/1
```

[0.2.6](https://github.com/HdrHistogram/hdr_histogram_erl/tree/0.2.6) seems to be fine though.

**Comment by darach** [comment]
Hi Sebastian.

Thanks for spotting and reporting this. It looks like some fixes to internals for better NIF resource
tracking has broken the print and other functions as a side effect. Use the 0.2.6 tag for now and I'll
fix the issues on master before dropping tag 0.2.7, which should be soon...

Cheers,

Darach.

**Comment by tisba** [comment]
Great. I also noticed btw, that the CSV export in elixir works fine, despite the comment and warning in `examples/simple.exs`. I'm looking forward to 0.2.7!

**Comment by darach** [comment]
Fixed in v0.3.0 but worth updating to v0.3.1 for some OOM / SIGSEG fixes.

**Comment by tisba** [comment]
Cool, thanks! I opened a PR (https://github.com/HdrHistogram/hdr_histogram_erl/pull/28) to fix the Elixir example.
